dev/push-setup/Gemfile

```diff
@@ -0,0 +1,8 @@
+source 'https://rubygems.org'
+
+gem 'berkshelf'
+gem 'test-kitchen'
+gem 'kitchen-vagrant'
+gem 'kitchen-nodes'
+gem 'chef-zero'
+gem 'chef'
```

Comment:
if we are depending on chedk, why to we need chef, etc?  Is this intended to be `bundle exec`d?